docs/payment-method-providers.md

```diff
@@ -6,29 +6,28 @@ The payment methods that can be tested without real money transactions have been
 
 `checkout-provider` values are provided too, but they are subject to change without notice.
 
-| Provider                                             | `checkout-provider`                                   | Credentials                                                             |
-| ---------------------------------------------------- | ----------------------------------------------------- | ----------------------------------------------------------------------- |
-| Nordea                                               | `nordea`                                              | Username: `123456`<br>Password: `1111`<br>Security code: any            |
-| OP                                                   | `osuuspankki`                                         | No credentials needed, will not present an UI on OP side                |
-| Handelsbanken<br>POP Pankki<br>Säästöpankki<br>OmaSP | `handelsbanken`<br>`pop`<br>`saastopankki`<br>`omasp` | Username: `11111111`<br>Password: `123456`<br>Security code: `123456`   |
```

Comment:
Not sure if you want to delete POP, OmaSP etc too?

Reply:
nope, reverted this change